client/app/templates/components/game-client.hbs

```diff
@@ -0,0 +1,33 @@
+<div class="container">
+  <div class="row">
+    <div class="col-md-12 console">
+      <div class="panel panel-default">
+        <div class="panel-heading">
+          <span class="glyphicon glyphicon-comment"></span> Client
+        </div>
+        <div class="panel-body">
+          <ul class="chat">
+            {{#each messages as |msg|}}
+              <li class="clearfix">
+                <div class="chat-body clearfix">
+                  {{#if msg.server}}
```

Comment:
If you go with my suggestion of the `cssClass` attribute then your template turns into something like:

```handlebars
<p class={{msg.cssClass}}>{{msg.text}}</p>
```